docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -611,6 +611,9 @@ By default, the vector values are not included. To include the vector values, se
 
 |===
 
+NOTE: Larger `limit` makes it more likely to find better results which are not found with smaller `limit`. Even the truly closest neighbour may be found only with sufficiently large `limit`.
+This can be unexpected if you try to compare the search results with different `limit`: one is not guaranteed to be the subset of another.
+The precision, especially with smaller `limit` values can be fine-tuned using hints.
```

Comment:
```suggestion
NOTE: Using a larger `limit` may yield better results than with a smaller `limit` — for example, the nearest neighbor may only be found with a sufficiently large `limit`. 
This can be unexpected if you are trying to compare search results that use a different `limit`, since one is not guaranteed to be a subset of another. 
For precision, especially with smaller `limit` values, use hints as a means of fine-tuning.
```